applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -281,6 +295,26 @@ static bool app_event_handler(const struct app_event_header *aeh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_CAF_BLE_SMP_TRANSFER_EVENTS) && is_ble_smp_transfer_event(aeh)) {
+		process_dvfs_states(DVFS_STATE_SMP_TRANSFER, true);
+		struct dvfs_state_timeout smp_transfer_timeout =
```

Comment:
That should be a pointer (as suggested here: https://github.com/nrfconnect/sdk-nrf/pull/16837#discussion_r1716437087). I am not even sure if it would work if you shallow-copy the structure here (delayable work is part of the structure).